h/api/search/index.py

```diff
@@ -23,25 +23,19 @@ def index(es, annotation, request):
     :param es: the Elasticsearch client object to use
     :type es: h.api.search.Client
 
-    :param annotation: the annotation to index
-    :type annotation: h.api.models.Annotation
+    :param annotation_dict: the presented annotation to index
+    :type annotation_dict: dict
 
     """
-    annotation_dict = presenters.AnnotationJSONPresenter(
-        request, annotation).asdict()
-
-    annotation_dict['target'][0]['scope'] = [
-        annotation.target_uri_normalized]
-
     es.conn.index(
         index=es.index,
         doc_type=es.t.annotation,
         body=annotation_dict,
-        id=annotation_dict["id"],
+        id=annotation_dict['id'],
     )
 
 
-def delete(es, annotation):
+def delete(es, annotation_dict):
```

Comment:
This could just take an id -- would save the cost of serializing and sending the whole thing through Rabbit.